### PR TITLE
Caoimhe/information service unit tests

### PR DESCRIPTION
**Pull request by CaoimheCahill**
Unit tests written for:

- Team
- Matches
- InformationService
- InformationController

Matches and Team Repository don't need a test class.
contextLoads commented out in InformationServiceTests as there is no database to test off at the moment

### CODE REVIEW

**Comment by CaoimheCahill** [comment]
> Logic looks good but unit tests are failing with something due to the .env file cannot be loacated. Would say its as we don't push this up and the github actions cannot find file . @CaoimhinHawk1 might need todo some stuff with the docker hub secret keys for this to work?

Could only fix by created test .env

File: .github/workflows/main_lastman-standing-app.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
 
+    env:
+        LMS_ENV_INFORMATION: ${{ secrets.LMS_ENV_INFORMATION}}
+
```
Comment (CaoimhinHawk1) [inline .github/workflows/main_lastman-standing-app.yml]: Just need to get rid of this from the Pull request and it should be fine, good work caoimhe

**Review by olanhealy — APPROVED** [review]
LGTM, good job on solving issue, mocking in unit tests can be difficult and you managed it well, approving

**Review by CaoimhinHawk1 — APPROVED** [review]
Good work Caoimhe, Essential work done GJ!

**Review by MarkHarrison03 — APPROVED** [review]
Looks good to me, tests are passing now. good work!

**Pull request merged**